src/psyclone/psyir/nodes/operation.py

```diff
@@ -224,38 +242,209 @@ def create(operator, lhs, rhs):
         child instances with optional names.
 
         :param operator: the operator used in the operation.
-        :type operator: \
+        :type operator:
             :py:class:`psyclone.psyir.nodes.BinaryOperation.Operator`
-        :param lhs: the PSyIR node containing the left hand side of \
-            the assignment, or a tuple containing the name of the \
+        :param lhs: the PSyIR node containing the left hand side of
+            the assignment, or a tuple containing the name of the
             argument and the PSyIR node.
-        :type lhs: Union[:py:class:`psyclone.psyir.nodes.Node`, \
+        :type lhs: Union[:py:class:`psyclone.psyir.nodes.Node`,
             Tuple[str, :py:class:`psyclone.psyir.nodes.Node`]]
-        :param rhs: the PSyIR node containing the right hand side of \
-            the assignment, or a tuple containing the name of the \
+        :param rhs: the PSyIR node containing the right hand side of
+            the assignment, or a tuple containing the name of the
             argument and the PSyIR node.
-        :type rhs: Union[:py:class:`psyclone.psyir.nodes.Node`, \
+        :type rhs: Union[:py:class:`psyclone.psyir.nodes.Node`,
             Tuple[str, :py:class:`psyclone.psyir.nodes.Node`]]
 
         :returns: a BinaryOperator instance.
         :rtype: :py:class:`psyclone.psyir.nodes.BinaryOperation`
 
-        :raises GenerationError: if the arguments to the create method \
+        :raises GenerationError: if the arguments to the create method
             are not of the expected type.
 
         '''
-        if not isinstance(operator, Enum) or \
-           operator not in BinaryOperation.Operator:
+        if (not isinstance(operator, Enum) or
+                operator not in BinaryOperation.Operator):
             raise GenerationError(
                 f"operator argument in create method of BinaryOperation class "
                 f"should be a PSyIR BinaryOperation Operator but found "
                 f"'{type(operator).__name__}'.")
-
         binary_op = BinaryOperation(operator)
         binary_op.addchild(lhs)
         binary_op.addchild(rhs)
         return binary_op
 
+    def _get_result_precision(self, precisions):
+        '''
+        Compares the two precisions to determine the precision of the result
+        of the operation.
+
+        If the two precisions are the same, then that value is returned.
+        Otherwise, Section 7.1.9.3 of the Fortran standard says that in this
```

Comment:
The Section numbers change for different versions of the standard, so you need to mention that this is for 2008 standard.

Reply:
Oops.